tasks/task08/test_shapes.py

```diff
@@ -0,0 +1,108 @@
+import unittest
+
+from tasks.task08.shapes import Circle, Point2D, Rectangle, Square, Shape2DCollection
+
+
+class TestShapes(unittest.TestCase):
```

Comment:
Best practices is separate test in different.
For example:
You have `Circle` and `Rectangle`, and good for another devs if you use `CircleTest` and `RectangleTest`, because `TestShapes` doesn't show anything or show just that this test for `Shapes`.